modules/EvseManager/Charger.hpp

```diff
@@ -187,9 +161,11 @@ class Charger {
         Disabled,
         Idle,
         WaitingForAuthentication,
+        PrepareCharging,
```

Comment:
Right now a call to Charger::cancelTransaction will actually not stop the transaction while in state PrepareCharging. This should be added to the check of currentState (https://github.com/EVerest/everest-core/blob/main/modules/EvseManager/Charger.cpp#L747) because it should be possible to stop a transaction while in PrepareCharging.

Reply:
Good catch, looks like you already fixed that

Reply:
The link above points to main branch